src/articles/frontend-debugging-101.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Frontend Debugging 101"
-date: "23-07-2024"
+date: "2024-07-23"
```

Comment:
this format change appears to have broken the dates in the /articles/page.tsx

could you update /lib/articles.ts to the new format?

Reply:
maintain it as yyyy-mm-dd, i prefer it like that 👍